### ISSUE TITLE
Can I use my own google home mini device with the control functionality

### ISSUE THREAD

**Issue by elblogbruno** [eouia/MMM-Assistant#72]
Can I use my own google home mini device with the control functionality?
I don't want to install de google assistant on the pi cause I have the google home mini

**Comment by eouia** [comment]
no, this module is made for Google Assistant embedding. So, it is not matched with your purpose.
